packaging/windows/buildpackage.sh

```diff
@@ -66,29 +65,24 @@ function makelauncher()
 
 function build_platform()
```

Comment:
This is now only called once, so it would be cleaner to inline this directly below.